client/src/components/TaskStatusChart/TaskStatusChart.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import styles from "./TaskStatusChart.module.css";
+import { PieChart, Pie, Cell, Legend } from "recharts";
+import { PropTypes } from "prop-types";
+
+const RADIAN = Math.PI / 180;
+const renderCustomizedLabel = ({
```

Comment:
I would rename this to `TaskStatusChart` or another name that is more fitting than the current one